charts/stable/games-on-whales/values.yaml

```diff
@@ -32,6 +32,10 @@ persistence:
     type: emptyDir
     mountPath: /home/retro
 
+# -- (object) Pass GPU resources to Xorg, steam and retroarch containers
+# See Custom configuration section in the Readme
+graphic_resources:
```

Comment:
I'm curious what you see as the benefit of adding this key over the "regular" `resources` section since these resources are not really different. Is it because of the `additionalContainer`?

Reply:
it is because it is needed in multiple containers and it is tedious to add to the (right) 3 ones.

The option of copy&paste of the additionalContainer also works.